CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/vips-role/tree/develop)
-
+- Update libvips source URL

```

Comment:
One more thingy, add your nickname at the end of the line and link the line with PR (as in [here](https://github.com/idealista/bitbucket-role/pull/5/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR7)) and that would be it.

Thanks!